### ISSUE TITLE
set view on Center Point

### ISSUE THREAD

**Issue by JitenNumberTank** [applm/CarouselWidget#12]
Is there any way to move selected view on center point? I hope I will get solution from your side ASAP.

**Comment by JitenNumberTank** [comment]
I have checked your Ma-Componant Library but that is not realiable for me. I want this functionality in this library. Could you set it in Carousel class?

**Comment by applm** [comment]
It is open source. Look at the code and add what you need,

**Comment by applm** [comment]
Or look at forks.. People already fixed lot of issues.

**Comment by JitenNumberTank** [comment]
Yes It open source, But if you have made code so you know better than me.
It will be good if you provide me changes

On Sat, May 14, 2016 at 3:19 PM, Martin Appl notifications@github.com
wrote:

> It is open source. Look at the code and add what you need,
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/applm/CarouselWidget/issues/12#issuecomment-219211348

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *

**Comment by JitenNumberTank** [comment]
Thanks for your reply. I have also looked in fork but couldn't found this
changes. Could you look at code and let me know.

On Sat, May 14, 2016 at 3:20 PM, Martin Appl notifications@github.com
wrote:

> Or look at forks.. People already fixed lot of issues.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/applm/CarouselWidget/issues/12#issuecomment-219211408

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *

**Comment by damienoliver** [comment]
Wow. Lazy AND entitled.

**Comment by applm** [comment]
I made it more than year ago. I wouldt actually have to spend same time reading code as you do. And I dont need that feature. You do.

**Comment by JitenNumberTank** [comment]
no. I am not lazy. I am also finding that. but couldn't found at there.
still looking in code

On Sat, May 14, 2016 at 3:23 PM, damienoliver notifications@github.com
wrote:

> Wow. Lazy AND entitled.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/applm/CarouselWidget/issues/12#issuecomment-219211522

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *

**Comment by JitenNumberTank** [comment]
Its not compulsury brother. but through your small help I can go forward

On Sat, May 14, 2016 at 3:24 PM, Jiten Ramoliya jiten@numbertank.com
wrote:

> no. I am not lazy. I am also finding that. but couldn't found at there.
> still looking in code
> 
> On Sat, May 14, 2016 at 3:23 PM, damienoliver notifications@github.com
> wrote:
> 
> > Wow. Lazy AND entitled.
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > https://github.com/applm/CarouselWidget/issues/12#issuecomment-219211522
> 
> ## 
> 
> ```
>        Thanks & Regards,
> ```
> 
> *Jiten Ramoliya *

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *

**Comment by JitenNumberTank** [comment]
I also provide you 5 start rating on this

On Sat, May 14, 2016 at 3:25 PM, Jiten Ramoliya jiten@numbertank.com
wrote:

> Its not compulsury brother. but through your small help I can go forward
> 
> On Sat, May 14, 2016 at 3:24 PM, Jiten Ramoliya jiten@numbertank.com
> wrote:
> 
> > no. I am not lazy. I am also finding that. but couldn't found at there.
> > still looking in code
> > 
> > On Sat, May 14, 2016 at 3:23 PM, damienoliver notifications@github.com
> > wrote:
> > 
> > > Wow. Lazy AND entitled.
> > > 
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly or view it on GitHub
> > > https://github.com/applm/CarouselWidget/issues/12#issuecomment-219211522
> > 
> > ## 
> > 
> > ```
> >        Thanks & Regards,
> > ```
> > 
> > *Jiten Ramoliya *
> 
> ## 
> 
> ```
>        Thanks & Regards,
> ```
> 
> *Jiten Ramoliya *

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *

**Comment by applm** [comment]
Scroling is handled in computeScroll method. It is moved by setScroll(). If you mean automatic centering with animation. My other deprecated repopsitory had this implemented. I think someone also talked here in comment about implementing it.

**Comment by JitenNumberTank** [comment]
Yes I mean automatic centering with animation. but that repository is
infinite looping. my requirement is not like that. Your current library
which I used is more suiteble for my code. If I set code in computeScroll
method then it stuck at there. don't allow me to scroll ahead.

final int screenCenter = getWidth() / 2 + getScrollX();

int childCenter = getChildCenter(mReverseOrderIndex);

int Offset = screenCenter - childCenter;

I am getting this offeset and also doing mScroller.startScroll(0, 0,
screenCenter - childCenter, 0, 1000);

but this not worked for me.

On Sat, May 14, 2016 at 9:25 PM, Martin Appl notifications@github.com
wrote:

> Scroling is handled in computeScroll method. It is moved by setScroll().
> If you mean automatic centering with animation. My other deprecated
> repopsitory had this implemented. I think someone also talked here in
> comment about implementing it.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/applm/CarouselWidget/issues/12#issuecomment-219227731

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *

**Comment by applm** [comment]
https://github.com/gintechsystems/Android-Carousel . Forked branch. It should have scroll snap according to author.

**Comment by JitenNumberTank** [comment]
Thanks For your help, Let me checkout code

On Mon, May 16, 2016 at 3:08 PM, Martin Appl notifications@github.com
wrote:

> https://github.com/gintechsystems/Android-Carousel . Forked branch. It
> should have scroll snap according to author.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/applm/CarouselWidget/issues/12#issuecomment-219386996

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *

**Comment by JitenNumberTank** [comment]
yes. This code is working as well. I am really grateful for  it. I will
give you start for your awesome project. thanks once again

On Mon, May 16, 2016 at 3:24 PM, Jiten Ramoliya jiten@numbertank.com
wrote:

> Thanks For your help, Let me checkout code
> 
> On Mon, May 16, 2016 at 3:08 PM, Martin Appl notifications@github.com
> wrote:
> 
> > https://github.com/gintechsystems/Android-Carousel . Forked branch. It
> > should have scroll snap according to author.
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > https://github.com/applm/CarouselWidget/issues/12#issuecomment-219386996
> 
> ## 
> 
> ```
>        Thanks & Regards,
> ```
> 
> *Jiten Ramoliya *

## 

```
       Thanks & Regards,
```

*Jiten Ramoliya *
